shared/config/config.go

```diff
@@ -103,6 +103,17 @@ type ChainConfig struct {
 	// Maximum total gas cost for rebalancing txs per chain, fails if the sum
 	// of rebalancing txs in UUSDC exceeds this threshold
 	MaxRebalancingGasCostUUSDC string `yaml:"max_rebalancing_gas_cost_uusdc"`
+	// FundRebalancingTimeout specifies how long to delay a rebalancing transfer when
+	// gas costs exceed MaxRebalancingGasCostUUSDC. After this timeout expires, the
+	// transfer will proceed if gas costs are below FundRebalancingCostCapUUSDC.
+	// Set to -1 to disable the timeout.
+	FundRebalancingTimeout *time.Duration `yaml:"profitable_relay_timeout"`
```

Comment:
if setting to -1 will disable the timeout, what is the reason to have this as a pointer? a 0ns timeout doesn't really make sense so we could probably just enforce in the config validation that the timeout isn't 0 (which will be the 0 value if it isn't a pointer), instead of having it as a pointer and checking that it is not nil

Reply:
this would also fix my above comment about the dereference before the nil check